cypher-shell/src/main/java/org/neo4j/shell/cli/InteractiveShellRunner.java

```diff
@@ -22,42 +25,51 @@
 import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import static org.neo4j.driver.internal.messaging.request.MultiDatabaseUtil.ABSENT_DB_NAME;
+import static org.neo4j.shell.DatabaseManager.DEFAULT_DEFAULT_DB_NAME;
+
 /**
  * A shell runner intended for interactive sessions where lines are input one by one and execution should happen
  * along the way.
  */
 public class InteractiveShellRunner implements ShellRunner, SignalHandler {
     static final String INTERRUPT_SIGNAL = "INT";
-    private final static AnsiFormattedText freshPrompt = AnsiFormattedText.s().bold().append("neo4j> ");
-    private final static AnsiFormattedText continuationPrompt = AnsiFormattedText.s().bold().append("       ");
-    private final static AnsiFormattedText transactionPrompt = AnsiFormattedText.s().bold().append("neo4j# ");
+    private final static String freshPrompt = "> ";
+    private final static AnsiFormattedText continuationPrompt = AnsiFormattedText.s().bold().append("  ");
```

Comment:
can't we use capital letters for statics?

Reply:
Sure